### ISSUE TITLE
Urgent highlighting method not working as intended for Tasklist widget

### ISSUE THREAD

**Issue by purhan** [qtile/qtile#2171]
# Issue description
Urgent text in tasklist is not working as intended. Below is a snippet from my configuration:
```python
        bottom=bar.Bar(
            [
                widget.TaskList(
                    borderwidth=0,
                    spacing=0,
                    highlight_method="block",
                    background=BLACK,
                    border=GREY,
                    urgent_alert_method="text",
                    urgent_text=RED,
                    markup_floating="<i>{}</i>",
                    markup_minimized="<s>{}</s>",
                    icon_size=0,
                    margin=0,
                    title_width_method="uniform",
                ),
            ],
            18,
```
As you can see, `urgent_alert_method` is text, however, `highlight_method` is block.

Ideally, `urgent_alert_method` should be the deciding factor for urgent clients.

### Result
When a window is urgent, `block` method is used for highlighting.
![image](https://user-images.githubusercontent.com/10897423/106393790-0eccd200-641f-11eb-81b3-1fa6558d968c.png)

### Expected
The opposite should've happened. The text should be red and block should be grey.

# Qtile version
`v0.16.1` as well as master (d68819e6f38a61cf43bbab3394ae0259ed367696).

**Comment by elParaguayo** [comment]
Yes - can see how that's not quite working in the code for that widget. 

Are you able to submit a PR or do you need someone else to do it?

It also looks like there's no `urgent_text` parameter for this one and it should be setting the text colour from the `urgent_border` parameter. Even still, it seems that the `highlight_method` defines the behavior.

**Comment by Purhan** [comment]
> It also looks like there's no urgent_text parameter for this one

I see. That should be added for consistency with other widgets (like `GroupBox`)

> Are you able to submit a PR or do you need someone else to do it?

Can't work on this right now. If someone else wants to work on it, please go ahead.
If nobody does, I will try working on it in a few days.

**Comment by m-col** [comment]
Presumably this is still an issue?

**Comment by elParaguayo** [comment]
@ramnes - this one's on my to do list!

**Comment by ramnes** [comment]
Now it is 😀
